### DISCUSSION TITLE
Conversational "Jan"?

### DISCUSSION THREAD

**Discussion by jmellinger473** [janhq/jan#1826] [General]
Question are there any plans to make "Jan" conversational? I have been playing with "Replica" but I want full control and run on my local Linux Mint computer.  EX: I wear a headset with a mic and ask "Hey Jan what's the weather in New York? And Jan responce " It blowing freaking snow!"

I'm currently running Jan using the Trinity LM.

**Comment by TomLucidor** [comment]
It can already do that by tweaking the settings and picking models specifically for conversation
